.travis.yml

```diff
@@ -12,7 +12,7 @@ cache:
   directories:
     - travis_phantomjs
 before_install:
-  - "npm install -g npm@4.1.2"
+  - "npm install -g npm@5.3.0"
```

Comment:
Previously we were shrink-wrapping our dependencies, which (I believe) required `npm 4.x`. Since you're using `npm 5`, you might want to commit the `package-lock.json` file.

@robertjd: What do you think?

Reply:
I'm inclined to add this later, after the next round of work (packages are going to change a lot)